src/fileSystemAssetLoader.js

```diff
@@ -15,7 +15,12 @@ export default class FileSystemAssetLoader {
     return new Promise((resolve, reject) => {
       const options = this.options;
       const buildDir = options.buildDir;
-      const mountPath = `${options.mountPath || ''}/`;
+
+      // Setup the mountPath, and make sure it ends with one '/'
```

Comment:
> it ends with one '/'
i think the comment is not accurate if options.mountPath ends with two '/' the code will not remove from it, so we can actually end up with more than one '/' at the end

Reply:
@mfazekas Thanks for the review!  I've updated the comment to be more accurate.